### PR TITLE
Use a flag for a test namespace

### PR DESCRIPTION
**Pull request by kyessenov**
Checks that Manager and Proxy can operate in a multi-namespace cluster.

---
This change is [Reviewable](https://reviewable.io/reviews/istio/manager/63)

### CODE REVIEW

File: test/integration/driver.go [hunk 1]
```diff
@@ -215,18 +204,33 @@ func makeRequests(pods map[string]string) map[string][]string {
...
-					match := regexp.MustCompile("X-Request-Id=(.*)").FindStringSubmatch(request)
-					if len(match) > 1 {
-						id := match[1]
-						log.Printf("id=%s\n", id)
-						out[src] = append(out[src], id)
-						out[dst] = append(out[dst], id)
-					} else if src != "t" || dst != "t" {
-						log.Fatalf("Failed to inject proxy from %s to %s (url %s)", src, dst, url)
+					n := 1
+					for {
```
Comment (ayj) [inline test/integration/driver.go]: The initialization and increment steps could be pulled into the for clause, i.e. "for (n := 1; ; n++)"

File: test/integration/driver.go [hunk 2]
```diff
@@ -236,14 +240,15 @@ func makeRequests(pods map[string]string) map[string][]string {
 	return out
 }
 
+// checkAccessLogs searches for request ids in the access logs
 func checkAccessLogs(pods map[string]string, ids map[string][]string) {
 	n := 0
```
Comment (ayj) [inline test/integration/driver.go]: Similar comment to line 207 re: pulling in 'n' initialization and increment into the for clause.

File: test/integration/driver.go (hunk 1 above)
Comment (kyessenov) [inline test/integration/driver.go]: Done

File: test/integration/driver.go (hunk 2 above)
Comment (kyessenov) [inline test/integration/driver.go]: Done

**Pull request merged**